googler

```diff
@@ -351,15 +360,13 @@ if duration is not None:
 baseurl = url
 basestart = start
 
-if debug:
-    print("[DEBUG] Base URL [%s]" % url)
+logger.debug("Base URL [%s]", url)
```

Comment:
Relating to the earlier comment, I think the `debug` flag is still useful and we can retain it everywhere.